packages/ts-transformers/package.json

```diff
@@ -56,8 +56,10 @@
     }
   },
   "dependencies": {
-    "ts-clone-node": "^3.0.0",
-    "typescript": "~5.5.0"
+    "ts-clone-node": "^3.0.0"
+  },
+  "peerDependencies": {
+    "typescript": "^5.5.0"
```

Comment:
Hmm... my big concern here is that because TypeScript doesn't follow semver, they could ship a change in 5.8 that breaks this library. I would rather specify a known-good range, but that would mean we would need to update the package to support every new typescript version.

Should we just do `>=5.5.0` and live with the inevitable future breakage?

Reply:
since the point here is for users to bring their own version of TS, i think being too restrictive does more harm than good. `^` seems like a reasonable constraint, with TS having majors roughly 2.5 years or so.